Automattic/wp-calypso
# Reader: Add time to read

Related to Automattic/loop#259

## Proposed Changes

Enriches the posts in the Reader state to have a `minutes_to_read` property

<img width="1062" alt="Screenshot 2024-12-16 at 2 45 57 PM" src="https://github.com/user-attachments/assets/fab5745f-9427-48f3-992a-3fdcbece36b3" />
 

## Why are these changes being made?


pe7F0s-2mQ-p2

## Testing Instructions



* Navigate to `/read`
* Inspect the Redux store for Reader posts and make sure the `minutes_to_read` field is present
* Make sure the `minutes_to_read` value doesn't seem wildly off

## Pre-merge Checklist
